sources/packages/backend/apps/db-migrations/src/main.ts

```diff
@@ -13,7 +13,7 @@ import { ormConfig } from "./data-source";
     await dataSource.query(`SET search_path TO ${ormConfig.schema}, public;`);
     await dataSource.query(`SET SCHEMA '${ormConfig.schema}';`);
     console.info(`**** Running migration ****`);
-    await dataSource.runMigrations();
+    await dataSource.runMigrations({ transaction: "each" });
```

Comment:
What does `{ transaction: "each" }` mean?

Reply:
Assuming that there are multiple migrations to be executed to have the DB updated, `each` will execute all the commands of the same migration file in a single transaction committing it every time.
Out-of-box behavior would be, to open a transaction, execute all migrations, and commit them all.